docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
-  var one = new Job("one", "alpine:3.4")
-  var two = new Job("two", "alpine:3.4")
-
-  one.tasks = ["echo world"]
-  one.run().then( result => {
-    two.tasks = ["echo hello " + result.toString()]
-    two.run().then( result2 => {
-      console.log(result2.toString())
-    })
-  })
-})
+Using default tag: latest
+latest: Pulling from library/busybox
+24fb2886d6f6: Pulling fs layer
+24fb2886d6f6: Verifying Checksum
+24fb2886d6f6: Download complete
+24fb2886d6f6: Pull complete
+Digest: sha256:52f73a0a43a16cf37cd0720c90887ce972fe60ee06a687ee71fb93a7ca601df7
+Status: Downloaded newer image for busybox:latest
+docker.io/library/busybox:latest
 ```
-[brigade-17.js](../../examples/brigade-17.js)
 
-Our actual containers (`one` and `two`) are not doing much work here. They're just
-echoing content to their standard output. But that information is captured by Brigade
-and sent back as a `Result` object.
+Note we could also take a look at the sidecar container logs on the job like so:
 
-- Job `one` returns `world`.
-- Job `two` takes that, and appends it to `echo hello`, which returns `hello world`
-- So `result2` will have a `Result` with `hello world`
+```console
+$ brig event logs --id 94d0fcd5-61dc-49be-bb81-3e5784e66a4a --job dind --container docker
 
+time="2021-09-23T19:00:11.230907700Z" level=info msg="Starting up"
+...
 ```
-Started brigade-worker-01bsy7k5h6n65gtt5sfrstte99-master
-yarn start v0.27.5
-$ node prestart.js
-prestart: src/brigade.js written
-$ node --no-deprecation ./dist/src/index.js
-Creating PVC named brigade-worker-01bsy7k5h6n65gtt5sfrstte99-master
-looking up default/github-com-brigadecore-empty-testbed-one
-Creating secret one-1505326897996-master
-Creating Job Cache PVC github-com-brigadecore-empty-testbed-one
-Creating pod one-1505326897996-master
-Timeout set at 900000
-default/one-1505326897996-master phase Pending
-default/one-1505326897996-master phase Pending
-default/one-1505326897996-master phase Succeeded
-looking up default/github-com-brigadecore-empty-testbed-two
-Creating secret two-1505326904108-master
-Creating pod two-1505326904108-master
-Creating Job Cache PVC github-com-brigadecore-empty-testbed-two
-Timeout set at 900000
-default/two-1505326904108-master phase Pending
-default/two-1505326904108-master phase Succeeded
-hello world
-
-beforeExit 0
-after handler is cleaning up build storage for brigade-worker-01bsy7k5h6n65gtt5sfrstte99-master
-exit called
-Done in 11.20s.
-```
-
-It is a good idea to call `toString()` on the `Result` object, since there is otherwise
-no guarantee about the type of data returned.
 
-With carefully constructed containers, you can get sophisticated and send structured
-data like JSON from one job to another. But remember that what is captured is the
-standard output (STDOUT) of the job, which is often where log data will also be sent.
-Sometimes it makes more sense to write structured files to the shared storage instead.
+### Accessing the host Docker socket
```

Comment:
Given the increasing prominence of containerd on k8s nodes, which unlike Docker cannot be used to build images anyway, do you have any thoughts on whether we should just _cut_ this feature from 2.0 entirely?

Given that we would always encourage DinD over DooD and given that we might even encourage Kaniko (or similar) over DinD (and DooD), it's not as if there's any shortage of safer alternatives.

lmk what you think.

Reply:
I'm up for at least creating a ticket to track and discuss the proposal to cut it out 👍

Reply:
> I'm up for at least creating a ticket to track and discuss the proposal to cut it out

That sgtm. I'll do that. If we do it, it's a rare post-beta breaking change, but I think pre-GA, security is certainly defensible grounds for such a change.